src/main/java/com/ncc/aif/ValidateAIFCli.java

```diff
@@ -72,8 +73,12 @@
     private static final Logger logger = (Logger) (org.slf4j.LoggerFactory.getLogger(Logger.ROOT_LOGGER_NAME));
     // ViolationThreshold
     private static final String ABORT_PARAMETER_STRING = "Abort parameter";
-    private static final int DEFAULT_MAX_VIOLATIONS = -1;
```

Comment:
Why not just have made DEFAULT_MAX_VIOLATIONS = Integer.MIN_VALUE?

Reply:
I slightly changed the context of these constants to make it work more consistently with `--depth`.  The `DEFAULT_*` and `MINIMUM_*` constants now refer to the default and minimum values (respectively) of the `[num]` parameter for these switches.  They no longer convey anything about whether the feature is enabled or not.

In fail-fast, the default and the minimum are the same (3), so I use the minimum constant.  It could be clearer or more readable/consistent if I set both `MINIMUM_MAX_VIOLATIONS` and `DEFAULT_MAX_VIOLATIONS` to 3.

That said, there _could_ be an additional constant set to `Integer.MIN_VALUE` to use for detecting whether or not the flag was specified...maybe called `UNSPECIFIED_VALUE`.  WDYT?

Reply:
I don't really have a strong opinion either way. Your reasoning is sound. Marking resolved.